languages/fi/Laminas_Captcha.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-i18n-resources for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-i18n-resources/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-i18n-resources/blob/master/LICENSE.md New BSD License
+ */
+
+/**
+ * EN-Revision: 30.Jul.2011
+ */
+return [
+    // Laminas\Captcha\ReCaptcha
+    "Missing captcha fields" => "Captchan kenttiä puuttuu",
+    "Failed to validate captcha" => "Captchan tarkistus epäonnistui",
+    "Captcha value is wrong: %value%" => "Captchan arvo %value% on väärä",
```

Comment:
The error message is wrong: 

```php
protected $messageTemplates = [
    self::MISSING_VALUE => 'Missing captcha fields',
    self::ERR_CAPTCHA   => 'Failed to validate captcha',
    self::BAD_CAPTCHA   => 'Captcha value is wrong',
];
```

https://github.com/laminas/laminas-captcha/blob/59173ecabf7602492345a39912874a9dafcacad1/src/ReCaptcha.php#L56-L60